packages/plugins/typescript/resolvers/src/index.ts

```diff
@@ -76,7 +76,18 @@ export type Resolver${capitalizedDirectiveName}WithResolve<TResult, TParent, TCo
   }
 
   const transformedSchema = config.federation ? addFederationReferencesToSchema(schema) : schema;
-  const visitor = new TypeScriptResolversVisitor({ ...config, directiveResolverMappings }, transformedSchema);
+
+  let visitor;
+
+  if (config.customVisitor) {
+    // @TODO: figure out how to use custom-visitor#VisirClass syntax
+    // instead of using default
+    const CustomVistor = require(config.customVisitor).default;
```

Comment:
I need a little guidance on how I can load the supplied string, in the same way mappers or resolverFn loads external imports!

Reply:
For loading modules correctly on both ESM and CJS the following is recommended: https://github.com/dotansimha/graphql-code-generator/blob/25c76c110999fe985b59bcb8e353edcec49ac0b9/packages/graphql-codegen-cli/src/codegen.ts#L33-L52

Reply:
I was looking for the function that loads strings like this:

`./my-custom-visitor#VisitorClass`

Reply:
@n1ru4l, what I'm trying is using `parseMapper` so that the syntax for configuring an import stays the same.

It'll look a little like this:

```ts
import { ExternalParsedMapper, parseMapper } from '@graphql-codegen/visitor-plugin-common';

    const parsedVisitorString = parseMapper(config.customVisitor) as ExternalParsedMapper;
```

This way you can specify the name of the class, rather than have to specifically use the default export.

It also looks like the `makeDefaultLoader` is in the codegen-cli package, should I duplicate it in this one? I'll need to spend a bit more time on this, but hopefully async imports are possible in a visitor!